cmd/kyma/alpha/init/module/module.go

```diff
@@ -40,11 +40,9 @@ The name of the  module directory is the same as the name of the module.
 Module name must start with a letter and may only consist of alphanumeric and '.', '_', or '-' characters.
 In the module directory, you'll find the template files and subdirectories corresponding to the required module structure:
     charts/       // folder containing a set of charts (each in a subfolder)
-    crds/         // folder containing all CRDs required by the module
     operator/     // folder containing the operator needed to manage the module
```

Comment:
should this be removed as well?

Reply:
No, the operator folder stays in the layout to store the operator(s) source code. It was removed from the layering so it will not be present in the descriptor and image.